src/components/textbox/textbox.style.ts

```diff
@@ -2,16 +2,15 @@ import styled, { css } from "styled-components";
 
 const ErrorBorder = styled.span`
   ${({ warning }: { warning: boolean }) => css`
+    width: 2px;
```

Comment:
comment: if we do get tokens to use lets leave it here as we don't have a legacy fallback